Week3/homework/js-exercises/creditCardProject.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+function validateCreditNumber(num) {
+    const array = [];
+    for(let i = 0; i < num.length; i++){
+        array.push(num[i]);
+    }
+
+    //At least two diffrent numbers should be represented
+    function similarity(array){
+            let firstItem = array[0]
+            let x = array.filter(elements => elements == firstItem).length != array.length ? true : false;
+            return x 
+        }
+
+        //The Sum of all the numbers must be greater than 16:
+        let sum = 0
+        for (let i = 0; i < array.length; i++){
+                let arrayInt = parseInt(array[i])
+                sum += arrayInt
+            }
+
+    switch(false){
```

Comment:
I don't understand why you use a `switch` statement with `false` in it. Can you please explain?